examples/01-transient_analyses/03-mode_superposition_transient.py

```diff
@@ -0,0 +1,140 @@
+"""
+.. _ref_msup_transient:
+
+Mode superposition transient analysis
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This example shows how to postprocess a mode superposition transient result
+and visualize the outputs. It also shows how to select modes for the modal expansion.
+
+"""
+# Import the necessary modules
+from ansys.dpf import core as dpf
+from ansys.dpf.core import examples
+
+###############################################################################
+# Download the mode superposition transient result example. This example is
+# not included in DPF-Core by default to speed up the installation.
+# Downloading this example should take only a few seconds.
+#
+# Next, create the model and display the state of the result. This mode superposition transient
+# result file contains several individual results, each at a different timestamp, automatically
+# expanded on all available modes.
+
+# msup_transient_files = examples.find_msup_transient()
+msup_transient_files = {
+    "rst": r"D:\ANSYSDev\Sandbox\UnitTestDataFiles\expansion\msup\Transient\plate1\file.rst",
+    "rdsp": r"D:\ANSYSDev\Sandbox\UnitTestDataFiles\expansion\msup\Transient\plate1\file.rdsp",
+    "mode": r"D:\ANSYSDev\Sandbox\UnitTestDataFiles\expansion\msup\Transient\plate1\modal\file.mode",
+    "modal_rst": r"D:\ANSYSDev\Sandbox\UnitTestDataFiles\expansion\msup\Transient\plate1\modal\file.rst",
+}
+# print(msup_transient_files)
+
+###############################################################################
+# Modal superposition on all modes available
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+# Create a datasource with upstream modal information
+data_sources = dpf.DataSources(msup_transient_files["rdsp"])
+
+up_stream_data_sources = dpf.DataSources(msup_transient_files["mode"])
+up_stream_data_sources.add_file_path(msup_transient_files["modal_rst"])
+data_sources.add_upstream(up_stream_data_sources)
+
+# Load into a model
+model = dpf.Model(data_sources)
+print(model)
+
+###############################################################################
+# Get the expanded displacement fields for each time-step
+disp = model.results.displacement.on_all_time_freqs.eval()
+print(disp)
+
+###############################################################################
+# Animate the result
+disp.animate(scale_factor=5.0)
+
+###############################################################################
+# Get the expanded displacement fields on selected time-steps:
+disp = model.results.displacement.on_time_scoping([1, 2, 3]).eval()
+print(disp)
+
+###############################################################################
+# Get the expanded displacement fields on part of the mesh, for selected time-steps:
+# TODO: NOT WORKING
+# partial_scoping = dpf.mesh_scoping_factory.nodal_scoping(
```

Comment:
what is not working?

Reply:
@cbellot000 when running this commented part I get:
```
Traceback (most recent call last):
  File "D:\ANSYSDev\Sandbox\pydpf-core\examples\01-transient_analyses\03-mode_superposition_transient.py", line 72, in <module>
    ).eval()
  File "D:\ANSYSDev\Sandbox\pydpf-core\src\ansys\dpf\core\results.py", line 266, in eval
    fc = self.__call__().outputs.fields_container()
  File "D:\ANSYSDev\Sandbox\pydpf-core\src\ansys\dpf\core\outputs.py", line 75, in __call__
    return self.get_data()
  File "D:\ANSYSDev\Sandbox\pydpf-core\src\ansys\dpf\core\outputs.py", line 72, in get_data
    return self._operator.get_output(self._pin, type_output)
  File "D:\ANSYSDev\Sandbox\pydpf-core\src\ansys\dpf\core\dpf_operator.py", line 494, in get_output
    parameters = {type_tuple[2]: type_tuple[1](self, pin)}
  File "D:\ANSYSDev\Sandbox\pydpf-core\src\ansys\dpf\gate\generated\operator_capi.py", line 402, in operator_getoutput_fields_container
    res = capi.dll.Operator_getoutput_FieldsContainer(op._internal_obj if op is not None else None, utils.to_int32(iOutput), ctypes.byref(utils.to_int32(errorSize)), ctypes.byref(sError))
OSError: exception: access violation reading 0x0000000000000008

ANSYS LICENSE MANAGER ERROR:

System error: 10093 "Either the application has not called WSAStartup, or WSAStartup failed.
".


Process finished with exit code 1
```

I think the `ANSYS LICENSE MANAGER ERROR` is new though

Reply:
who looks bad, did you create a bug in TFS to track it?

Reply:
Not yet, I'll do it right now